### ISSUE TITLE
Another error with particular TBI file

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/tabix-js#17]
Here is a snippet of a vcf with a failing tabix file

[failing_tabix.tar.gz](https://github.com/GMOD/tabix-js/files/2920145/failing_tabix.tar.gz)

Ends up getting a negative chunkCount

**Comment by cmdcolin** [comment]
@garrettjstevens i'll go ahead and take a look at this

**Comment by cmdcolin** [comment]
The reason this was happening was due to chromosomes of length greater than 2^29 being indexed which is technically a tabix limit but older versions of tabix allowed it

Tabix 1.2.1 allows it but tabix 1.9 gives a proper error

```
[E::hts_idx_push] Region 536930580..536930581 cannot be stored in a tbi index. Try using a csi index with min_shift = 14, n_lvls >= 6
tbx_index_build failed: G3116.vcf.gz
```

**Comment by cmdcolin** [comment]
Possible fix in #18
